base/applications/network/net/main.c

```diff
@@ -45,6 +46,24 @@ COMMAND cmds[] =
 
 HMODULE hModuleNetMsg = NULL;
 
+VOID PrintTranslatedStatus(
+    NET_API_STATUS status)
+{
+    switch (status)
+    {
+        case NERR_Success:
+            ConResPuts(StdOut, IDS_SUCCESS);
+            break;
+        case ERROR_ALIAS_EXISTS:
+           ConResPuts(StdOut, IDS_USEREXISTS);
+           break;
+        case NERR_UserNotFound:
+            ConResPuts(StdOut, IDS_NOTFOUND);
```

Comment:
```suggestion
            ConResPuts(StdErr, IDS_NOTFOUND);
```

Reply:
Note that handling this error separately seems to duplicate the error output you may obtain from the PrintMessageString call with this `NERR_UserNotFound` status value. Are you sure Windows' net.exe utility also does a separate handling of this error?